indico/core/config.py

```diff
@@ -208,6 +210,9 @@ def load_config(only_defaults=False, override=None):
         data['CONFIG_PATH_RESOLVED'] = resolved_path
         if resolved_path is not None:
             data['LOGGING_CONFIG_PATH'] = os.path.join(os.path.dirname(resolved_path), data['LOGGING_CONFIG_FILE'])
+        if data['LOGO_URL'] is not None:
+            data['ABSOLUTE_LOGO_URL'] = data['LOGO_URL'] if data['LOGO_URL'].startswith('http') else \
+                f"{data['BASE_URL']}{data['LOGO_URL']}"
```

Comment:
This is not the right place. Either make it a property on the `Config` class, or do it in `_postprocess_config`.

Also, the logic can be greatly simplified - `urljoin` is smart enough to handle all the cases:

```suggestion
        data['ABSOLUTE_LOGO_URL'] = urljoin(data['BASE_URL'], data['LOGO_URL']) if data['LOGO_URL'] else None
```